datascrubber/cli.py

```diff
@@ -0,0 +1,300 @@
+                })
+            )
+            threads.append(thread)
+
+    elif args.postgresql_hosts is not None:
+        for host in args.postgresql_hosts:
+            thread = threading.Thread(
+                target=worker,
+                kwargs=({
+                    'dbms': 'postgresql',
+                    'hostname': host,
+                    'target_accounts': args.share_with,
+                })
+            )
+            threads.append(thread)
+
+    for thread in threads:
+        thread.start()
+
+    for thread in threads:
+        thread.join()
+
+    logger.info('All tasks completed')
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description='GOV.UK data scrubber'
+    )
+
+    mysql_selection = parser.add_mutually_exclusive_group()
+    mysql_selection.add_argument(
+        '--mysql-hosts',
+        required=False,
+        nargs='+',
+        type=str,
+        help='Hostnames of source MySQL instances or clusters'
+    )
+
+    mysql_selection.add_argument(
+        '--mysql-instances',
+        required=False,
+        nargs='+',
+        type=str,
+        help="RDS instance identifiers of source MySQL RDS instances or "
+             "clusters. If specified, supercedes --mysql-hosts."
+    )
+
+    mysql_selection.add_argument(
+        '--mysql-snapshots',
+        required=False,
+        type=str,
+        nargs='+',
+        help="Snapshot identifiers to use for MySQL. Defaults to the most "
+             "recent automatic snapshot for each instance or cluster. If "
+             "specified, supercedes --mysql-hosts and --mysql-instances."
+    )
+
+    postgresql_selection = parser.add_mutually_exclusive_group()
+    postgresql_selection.add_argument(
+        '--postgresql-hosts',
+        required=False,
+        nargs='+',
+        type=str,
+        help="Hostname of source Postgres instances or clusters"
+    )
+
+    postgresql_selection.add_argument(
+        '--postgresql-instances',
+        required=False,
+        nargs='+',
+        type=str,
+        help="RDS instance identifiers of source Postgres RDS instances or "
+             "clusters. If specified, supercedes --postgresql-hosts."
+    )
+
+    postgresql_selection.add_argument(
+        '--postgresql-snapshots',
+        required=False,
+        nargs='+',
+        type=str,
+        help="Snapshot identifiers to use for Postgres. Defaults to the most "
+             "recent automatic snapshot for each instance or cluster. If "
+             "specified, supercedes --postgresql-hosts and --postgresql-instances."
+    )
+
+    parser.add_argument(
+        '--log-level',
+        required=False,
+        type=str,
+        choices=['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG'],
+        default='INFO',
+        help="Log level (default: INFO)",
+    )
+
+    parser.add_argument(
+        '--log-mode',
+        required=False,
+        type=str,
+        choices=['console', 'syslog'],
```

Comment:
What's the default `--log-mode`?

Reply:
If stdout is a tty, it defaults to `console`, otherwise `syslog`.